test/simulate/simulateDruidSql.mocha.js

```diff
@@ -509,7 +509,7 @@ describe('simulate DruidSql', () => {
             sqlTimeZone: 'Etc/UTC',
           },
           query:
-            'SELECT\n(t.ip_prefix) AS "Ip_prefix",\nCOUNT(*) AS "count"\nFROM "diamonds" AS t\nWHERE IP_SEARCH(\'192.0\', IP_PREFIX_PARSE("ip_prefix"))\nGROUP BY 1\nORDER BY "count" DESC\nLIMIT 10',
+            'SELECT\n(t.ip_prefix) AS "Ip_prefix",\nCOUNT(*) AS "count"\nFROM "diamonds" AS t\nWHERE IP_SEARCH(\'192.0\', "ip_prefix")\nGROUP BY 1\nORDER BY "count" DESC\nLIMIT 10',
```

Comment:
Need to fix this to simulate with `IP_STRINGIFY` in splits.
```suggestion
            'SELECT\nIP_STRINGIFY(t.ip_prefix) AS "Ip_prefix",\nCOUNT(*) AS "count"\nFROM "diamonds" AS t\nWHERE IP_SEARCH(\'192.0\', "ip_prefix")\nGROUP BY 1\nORDER BY "count" DESC\nLIMIT 10',
```